AirCasting/SessionViews/SessionCartView.swift

```diff
@@ -56,7 +66,17 @@ struct SessionCellView: View {
     }
 }
 
-private extension SessionCellView {
+private extension SessionCartView {
+    
+    var header: some View {
+        SessionHeaderView(
+            action: {
+                withAnimation {
+                    isCollapsed = !isCollapsed
```

Comment:
you can use [toggle](https://developer.apple.com/documentation/swift/bool/2994863-toggle) here